seeds/tables/rops.js

```diff
@@ -1,6 +1,30 @@
 const rops = require('../data/rops');
 
+const defaults = {
+  basicSubpurposes: JSON.stringify([ 'oncology' ]),
+  endangered: false,
+  ga: false,
+  newGeneticLine: false,
+  nmbas: false,
+  otherSpecies: true,
+  placesOfBirth: JSON.stringify([ 'uk-licenced' ]),
+  postnatal: true,
+  proceduresCompleted: true,
+  productTesting: false,
+  purposes: JSON.stringify([ 'basic' ]),
+  reuse: false,
+  rodenticide: false,
+  scheduleTwoDetails: null,
+  species: JSON.stringify({ precoded: ['mice'], otherSpecies: [] })
+};
+
 module.exports = {
-  populate: knex => knex('rops').insert(rops),
+  populate: knex => knex('rops').insert(
+    rops.map(rop => ({
+      ...defaults,
+      submittedDate: rop.status === 'submitted' ? (new Date()).toISOString() : null,
```

Comment:
```suggestion
      submittedDate: rop.status === 'submitted' ? rop.submittedDate || (new Date()).toISOString() : null,
```

Reply:
That's a no-op since the submitted date is inherited from the `...rop` on the next line.

Reply:
doh, saw the `...defaults` and completely missed the `...rop`